deepspeed/runtime/engine.py

```diff
@@ -1616,6 +1700,17 @@ def deepspeed_io(self,
             data_parallel_world_size = self.mpu.get_data_parallel_world_size()
             data_parallel_rank = self.mpu.get_data_parallel_rank()
 
+        deepspeed_engine_config = {}
```

Comment:
Minor point, this is not really deepspeed engine config, but more precisely data loader config.

Reply:
Have changed it to deepspeed_dataloader_config